services/company_graph_store.py

```
@@ -1028,7 +1041,15 @@ async def _initialize_schema(self):
         await conn.execute("CREATE INDEX IF NOT EXISTS idx_specialists_company ON specialists(company_id)")
         await conn.execute("CREATE INDEX IF NOT EXISTS idx_knowledge_company ON knowledge_items(company_id)")
         await conn.execute("CREATE INDEX IF NOT EXISTS idx_detected_systems_company ON detected_systems(company_id)")
-        
+
+        # Migration: add the websites.data JSON-blob column to pre-existing DBs
+        # created before scan results were persisted. SQLite has no
+        # "ADD COLUMN IF NOT EXISTS", so guard on the failure.
+        try:
+            await conn.execute("ALTER TABLE websites ADD COLUMN data TEXT")
+        except Exception:
+            pass  # column already exists
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)